Dockerfile

```
@@ -29,11 +29,13 @@ ADD . /app
 
 
 # Sync dependencies
-RUN uv sync --frozen && uv sync --group prod --frozen
+RUN uv sync --all-groups --frozen
 
 # Collect static files
 RUN mkdir -p /app/staticfiles
 
 RUN chmod +x /app/entrypoint.sh
+RUN chmod +x /app/collectstatic.sh
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)